test/tools/integration/Makefile

```diff
@@ -1,3 +1,6 @@
+# prow uses BUILD_ID
+# TODO: remove CIRCLE_BUILD_NUM entirely after moving to prow
+CIRCLE_BUILD_NUM=$$BUILD_ID
```

Comment:
Right now this still must be `CIRCLE_BUILD_NUM ?=$$BUILD_ID` to only set if if `CIRCLE_BUILD_NUM` is unset

Reply:
Done.